python/composio/tools/toolset.py

```diff
@@ -628,6 +650,22 @@ def _process_schema_properties(self, action: Action, properties: t.Dict) -> t.Di
             type_="schema",
         )
 
+    def _merge_processors(self, processors: ProcessorsType) -> None:
```

Comment:
In the `_merge_processors` method, consider adding a type check for the `processors` parameter to ensure it's of type `ProcessorsType`. This will help catch potential errors early and improve type safety.